CalibMuon/DTCalibration/python/ALCARECODtCalib_Output_cff.py

```diff
@@ -40,3 +40,13 @@
 import copy
 OutALCARECODtCalib = copy.deepcopy(OutALCARECODtCalib_noDrop)
 OutALCARECODtCalib.outputCommands.insert(0, "drop *")
+
+## customizations for the pp_on_AA_2018 eras
+from Configuration.Eras.Modifier_pp_on_AA_2018_cff import pp_on_AA_2018
+
+OutALCARECODtCalibHI = copy.deepcopy(OutALCARECODtCalib_noDrop)
+OutALCARECODtCalibHI.outputCommands.insert(0, "drop *")
+OutALCARECODtCalibHI.outputCommands.append("keep *_hiSelectedVertex_*_*")
```

Comment:
Hi @tocheng I am wondering is this event product available with the`pp_on_AA_2018` era customisations?
@mandrenguyen

Reply:
@mmusich Thanks for pointing it out. I checked the configuration generated using DP config with pp_on_AA scenario, the hiSelectedVertex is not in the sequence. It should be the module in heavyIon typed reconstruction. I can remove it after the test is done.